sklearn/tests/test_multioutput.py

```diff
@@ -864,3 +864,41 @@ def test_multioutput_regressor_has_partial_fit():
     msg = "This 'MultiOutputRegressor' has no attribute 'partial_fit'"
     with pytest.raises(AttributeError, match=msg):
         getattr(est, "partial_fit")
+
+
+# TODO(1.8):  remove when deprecated `base_estimator` is removed
+@pytest.mark.parametrize("Estimator", [ClassifierChain, RegressorChain])
+def test_base_estimator_deprecation(Estimator):
+    # Check that FutureWarning is raised when calling .fit()
+    X = np.array([[1, 2], [3, 4]])
+    y = np.array([[1, 0], [0, 1]])
+
+    estimator = LogisticRegression(solver="liblinear")
+
+    with pytest.warns(FutureWarning):
+        chain = Estimator(base_estimator=estimator)
+        chain._validate_estimator()
+
+    with pytest.raises(ValueError):
+        chain = Estimator(base_estimator=estimator, estimator=estimator)
+        chain._validate_estimator()
+
+
+@pytest.mark.parametrize("Estimator", [ClassifierChain, RegressorChain])
+def test_validate_estimator(Estimator):
+    X = np.array([[1, 2], [3, 4]])
+    y = np.array([[1, 0], [0, 1]])
+
+    estimator = LogisticRegression(solver="liblinear")
+    chain1 = Estimator(estimator)
+    chain2 = Estimator()
+
+    # Test the _estimator attribute is set is each case
+    chain1._validate_estimator()
+    chain2._validate_estimator(estimator)
+    assert hasattr(chain1, "_estimator") and hasattr(chain2, "_estimator")
+    assert chain1._estimator == estimator and chain2._estimator == estimator
+
+    with pytest.raises(ValueError):
+        chain = Estimator()  # no estimator provided
+        chain._validate_estimator()
```

Comment:
We don't need this test because it checks some private API that is going to be tested with the public call.